hydra-tui/src/Hydra/TUI.hs

```diff
@@ -77,6 +77,7 @@ data State
   = Disconnected
       { nodeHost :: Host
       , now :: UTCTime
+      , hydraHeadId :: Maybe HeadId
```

Comment:
This should not be there. When disconnected we would not know whether the node tracks a head (and which id) or not.